com.gitlab.fossbr.WhatsAppQT.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.gitlab.fossbr.WhatsAppQT
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-21.08'
+command: WhatsAppQT
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=xdg-download"
+- "--device=dri"
+- "--env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess"
+- "--talk-name=org.kde.StatusNotifierWatcher"
+- "--own-name=org.kde.*"
```

Comment:
again, I was asking why this is necessary. "owning" the whole "org.kde.*" bus is excessively broad.

Reply:
ok, ill fix this.